frontends/web/src/routes/new-settings/about.tsx

```diff
@@ -0,0 +1,25 @@
+import { useTranslation } from 'react-i18next';
+import { Main, Header } from '../../components/layout';
+import { View, ViewContent } from '../../components/view/view';
+import { WithSettingsTabs } from './components/tabs';
+import { AppVersion } from './components/about/app-version-setting';
+
+type TProps = {
+    deviceIDs: string[]
+  }
+
+export const About = ({ deviceIDs }: TProps) => {
+  const { t } = useTranslation();
+  return (
+    <Main>
+      <div className="hide-on-small"><Header title={<h2>{t('sidebar.settings')}</h2>} /></div>
+      <View fullscreen={false}>
+        <ViewContent>
+          <WithSettingsTabs subPageTitle={'Appearance'} hideMobileMenu deviceIDs={deviceIDs}>
```

Comment:
subPageTitle should be About, please also add to en/app.json

Reply:
> subPageTitle should be About

Lol. True

> please also add to en/app.json

There are many that are still not there, I can start adding them I guess..